src/genomic_benchmarks_qc/report/classes_html_report.py

```
@@ -291,52 +307,167 @@ def generate_nucleotide_flags_html(summary_statuses, flag_prefix):
+    end_position, scored_end_position = position_windows(stats1, stats2)
+
+    if end_position < 1:
+        return ('<p>Sequences are too short for per-position '
+                'statistics, so no positions were analysed.</p>')
+
+    def coverage_at(position):
+        parts = []
+        for stats in (stats1, stats2):
+            label = stats.label if stats.label is not None else stats.filename
+            coverage = stats.coverage_at(position)
+            count = int(round(coverage * stats.stats['Number of sequences']))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)